.github/workflows/windows-ci.yml

```diff
@@ -0,0 +1,52 @@
+name: Windows CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+env:
+  SBT_OPTS: '-Xmx6g -Xms1g -Xss4m'
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        java: [ '8' ]
+
+    runs-on: windows-latest
+
+    steps:
+    - name: Set up git to use LF
+      run: |
+        git config --global core.autocrlf false
+        git config --global core.eol lf
+    - uses: actions/checkout@v3
+      with:
+        # fetch the Git tags for sbt-dynver and sbt-version-policy to work correctly
```

Comment:
Hum, we are not using these, are we?

Reply:
Oops, indeed. Copy-pasta from another repo where they are actually used.